.github/workflows/coverage.yml

```diff
@@ -0,0 +1,34 @@
+name: Coverage report
+
+on:
+  # to be removed
+  pull_request:
+    branches: [ main ]
+  # Only manual running
+  workflow_dispatch:
```

Comment:
Once we've tested that this works, shouldn't this run on pushes to `main`?

Reply:
Done